src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```diff
@@ -193,7 +440,13 @@ public void controlProperty(ControllableProperty controllableProperty) {
 	 */
 	@Override
 	public void controlProperties(List<ControllableProperty> controllableProperties) {
-
+		controllableProperties.forEach(p -> {
```

Comment:
controllableProperties can be null or empty, please check and throw an exception with a clear err msg